docs/docs/configuration/enum.mdx

```diff
@@ -21,14 +21,24 @@ Available strategies:
 
 The `IgnoreCase` property allows to opt in for case insensitive mappings (defaults to `false`).
 
+Enum from/to strings mappings can be customized by setting the enum naming strategy to be used.
+You can specify the naming strategy using `NamingStrategy` in `MapEnumAttribute` or `MapperAttribute`.
```

Comment:
and also in `MapperDefaultsAttribute` 😊

Reply:
Added support for MapperDefaultsAttribute